Owncloud iOs Client/Files/Edit/EditFileViewController.m

```diff
@@ -0,0 +1,206 @@
+//
+//  EditFileViewController.m
+//  Owncloud iOs Client
+//
+//  Created by Noelia Alvarez on 04/05/16.
+//
+//
+
+/*
+ Copyright (C) 2016, ownCloud, Inc.
+ This code is covered by the GNU Public License Version 3.
+ For distribution utilizing Apple mechanisms please see https://owncloud.org/contribute/iOS-license-exception/
+ You should have received a copy of this license
+ along with this program. If not, see <http://www.gnu.org/licenses/gpl-3.0.en.html>.
+ */
+
+#import "EditFileViewController.h"
```

Comment:
I think Customization.h, OCCommunication.h and ManageNetworkErrors.h are not used